pkg/repository/udmrepo/kopialib/lib_repo.go

```diff
@@ -384,7 +386,11 @@ func (kor *kopiaObjectReader) Read(p []byte) (int, error) {
 
 	n, err := kor.rawReader.Read(p)
 	if err != nil {
-		return 0, errors.Wrap(err, "error to read object")
+		if errors.Is(err, io.EOF) {
```

Comment:
Is it necessary to wrap other errors?

Reply:
True, I will remove the wrap for all the method of io.Reader, io.Writer, io.Closer and io.Seeker, these are the system wide standard errors, the caller may check these standard errors and wrapping it indeed brings us awkward

Reply:
Done